### ISSUE TITLE
[a11y-defect-2]: Issue accordion's contents should not have a tabindex of 0

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#60176]
### Point of contact

joshkimux

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

The contents of the issue accordions `<div id="content" tabindex="0"></div>` have a `tabindex="0"` which creates a new tab stop. 

This may be confusing for screen reader or keyboard users, as it will give it the impression that it is interactive even though it is not.

<img width="1492" alt="Screenshot 2023-06-12 at 3 36 25 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/8741d5d6-2fb6-4f58-b5f4-979195f6ebe5">
 

### Reproduction steps

Reproduction is spotty on staging for Claim Status Details. This should appear for any supplemental claim.

- Log in as user 233
- [View claim SC7416](https://staging.va.gov/track-claims/appeals/SC7416/detail), if that does not work, look for another supplemental claim
- Navigate to the issues tab
- Using the tab key, confirm you can tab to the contents of the "currently on review" accordion

### Proposed solution or next steps

- [ ] Conduct brief technical discovery into why `tabindex="0"` was added in the first place. 
- [ ] If there was no historical or functional reason for it, delete it.
- [ ] If there _is_ a historical or functional reason for this to receive focus...
  - [ ] Insert a `tabindex="-1"`
  - [ ] Post managing focus, remove the `tabindex="-1"`

### References, articles, or WCAG support

1. [Don't Use Tabindex Greater than 0](https://adrianroselli.com/2014/11/dont-use-tabindex-greater-than-0.html)

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by jacobworrell** [comment]
@joshkimux so it turns out that tableindex=0 code is injected into the component as part of the design system. You might want to punt this over to their team to take a look as we can see no reason for its existence on our end.

**Comment by jaredcunha** [comment]
It's hardcoded into the web component:
https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-accordion/va-accordion-item.tsx#L134

**Comment by joshkimux** [comment]
@jacobworrell @jaredcunha 😭 I did not expect this.
I'll mark this as blocked and open up a separate DST ticket to fix this as its source.

**Comment by jaredcunha** [comment]
This issue is now resolved by this merge: https://github.com/department-of-veterans-affairs/component-library/pull/730
